src/main/java/com/leverx/learningmanagementsystem/btp/registry/controller/SubscriptionController.java

```diff
@@ -0,0 +1,53 @@
+package com.leverx.learningmanagementsystem.btp.registry.controller;
+
+import com.leverx.learningmanagementsystem.btp.registry.model.DependenciesResponseDto;
+import com.leverx.learningmanagementsystem.btp.registry.model.SubscriptionRequestDto;
+import com.leverx.learningmanagementsystem.btp.registry.service.SubscriptionService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+import static org.springframework.http.HttpStatus.OK;
+
+@RestController
+@RequestMapping("/api/v1")
```

Comment:
maybe it would be even better to change path to `/api/v1/subscriptions`
```
PUT /api/v1/subscriptions/tenants/{id}
DELETE /api/v1/subscriptions/tenants/{id}
GET  /api/v1/subscriptions/dependencies
```